src/sentry/integrations/bitbucket/urls.py

```diff
@@ -4,12 +4,14 @@
 
 from .descriptor import BitbucketDescriptorEndpoint
 from .installed import BitbucketInstalledEndpoint
+from .uninstalled import BitbucketUninstalledEndpoint
 from .webhook import BitbucketWebhookEndpoint
 from .search import BitbucketSearchEndpoint
 urlpatterns = patterns(
     '',
     url(r'^descriptor/$', BitbucketDescriptorEndpoint.as_view()),
     url(r'^installed/$', BitbucketInstalledEndpoint.as_view()),
+    url(r'^uninstalled/$', BitbucketUninstalledEndpoint.as_view()),
```

Comment:
How does this URL get registered with bitbucket?

Reply:
@markstory in the descriptor https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/bitbucket/descriptor.py#L30

Reply:
:+1: thank you for explaining that.